subprojects/gradle/room-config/src/main/kotlin/com/avito/android/plugin/incremental/FakeProcessingEnvironment.kt

```diff
@@ -0,0 +1,28 @@
+package com.avito.android.plugin.incremental
+
+import java.util.Locale
+import javax.annotation.processing.Filer
+import javax.annotation.processing.Messager
+import javax.annotation.processing.ProcessingEnvironment
+import javax.lang.model.SourceVersion
+import javax.lang.model.util.Elements
+import javax.lang.model.util.Types
+
+internal class FakeProcessingEnvironment(
+    private val options: Map<String, String>
+) : ProcessingEnvironment {
+    override fun getElementUtils(): Elements? = null
+
+    override fun getTypeUtils(): Types? = null
+
+    override fun getMessager(): Messager = SystemOutMessenger()
+
+    override fun getLocale(): Locale = Locale.getDefault()
+
+    override fun getSourceVersion(): SourceVersion =
+        SourceVersion.RELEASE_8
```

Comment:
Should we get a real value from a project?

Reply:
I don't think so. It's a dummy processing environment.

Reply:
Not actual. Move this check from this PR to another task.